spec/SPEC.md

```diff
@@ -68,16 +59,15 @@ Next, an Input Manifest lists the Artifact Identifier of every input used to
 produce an artifact. For example, if an executable is compiled by linking
 together a collection of object files, the Artifact Identifier of every object
 file would be listed in the Input Manifest for the executable. Input Manifests
-can be identified by treating them as artifacts and applying the same identifier
-heuristic to them applied to any other artifact.  For purposes of discussion,
-these are typically called Input Manifest Idenftiers or Input Manifest IDs
-or IMIDs. The Input Manifest Identifier can be embedded directly into
-executable files, or can be provided in a separate file alongside the artifact
-whose inputs they describe.
+can be identified by their own Artifact Identifier. The Artifact ID for the
+manifest can be embedded directly into executable files, or can be provided
+in a separate file alongside the artifact whose inputs they describe.
```

Comment:
I have a slight concern that removing the IMID as a concept itself introduces confusion.  Given an artifact... one would like a simple way to distinguish conversationally between the artifact id of the artifact, and the artifact id of the input manifest of the artifact.  Artifact ID and Input Manifest ID (IMID) filled that role.  We introduced the concept because we had confusion downstream by communities like SPDX as to whether the artifact id field for an artifact should be the artifact id of the artifact or the artifact id of the artifacts input manifest.

Reply:
Possible idea: "Input Manifest ID" -> "Input Manifest AID"

"AID" being a shorthand for "Artifact ID". This would be a new shorthand, but hopefully makes it clearer that we're discussing the Artifact ID of the Input Manifest, and not some new _type_ of ID.